### ISSUE TITLE
Not build after last docker image push

### ISSUE THREAD

**Issue by ricardoapaes** [beevelop/docker-cordova#58]
On build image error show:

Failed to run "javac -version", make sure that you have a JDK version 8 installed.
You can get it from the following location:
https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html

Your JAVA_HOME is invalid: /usr/lib/jvm/java-8-openjdk-amd64

The same problem also happens in travis.

**Comment by aadityamundhalia** [comment]
I have solve this issue
Please follow along:
apt-get install openjdk-8-jdk
java -version
apt-get install wget
wget https://services.gradle.org/distributions/gradle-5.0-bin.zip -P /tmp;
apt-get install -y unzip
unzip -d /opt/gradle /tmp/gradle-*.zip
ls /opt/gradle/gradle-5.0
apt-get install -y nano
nano /etc/profile.d/gradle.sh
paste below in the file
_export GRADLE_HOME=/opt/gradle/gradle-5.0_
_export PATH=${GRADLE_HOME}/bin:${PATH}_

chmod +x /etc/profile.d/gradle.sh
source /etc/profile.d/gradle.sh
gradle -v

cordova requirements

Should pass now.

**Comment by joelambert** [comment]
Also getting this issue when using on our CI.

Is there any update on how to fix?

**Comment by ricardoapaes** [comment]
> Also getting this issue when using on our CI.
> 
> Is there any update on how to fix?

Sorry @beevelop,

But when the error occurred I had an image on my local machine without the error and uploaded it to the docker hub at that address:

docker pull likesistemas/cordova:stable

**Comment by mikebridge** [comment]
I think the jdk is getting removed during the various calls to `apt-get purge`, at least when I try to build it on top of Ubuntu 20:

```
apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false
Reading package lists...
Building dependency tree...
Reading state information...
The following packages will be REMOVED:
  ...
  openjdk-8-jdk*
  openjdk-8-jdk-headless* openjdk-8-jre* openjdk-8-jre-headless* 
   ...
0 upgraded, 0 newly installed, 273 to remove and 0 not upgraded.
After this operation, 752 MB disk space will be freed.
(Reading database ... 26832 files and directories currently installed.)
...
Removing openjdk-8-jdk:amd64 (8u265-b01-0ubuntu2~20.04) ...
Removing openjdk-8-jre:amd64 (8u265-b01-0ubuntu2~20.04) ...
Removing openjdk-8-jdk-headless:amd64 (8u265-b01-0ubuntu2~20.04) ...
```

**Comment by beevelop** [comment]
Fixed with `v2021.02.1`; the upstream image has been reworked to prevent this from happening. ✅
